src/main/scala/cognite/spark/v1/package.scala

```diff
@@ -3,9 +3,13 @@ package cognite.spark
 import com.cognite.sdk.scala.common.{NonNullableSetter, SdkException, SetNull, SetValue, Setter}
 import com.cognite.sdk.scala.v1.{SequenceColumn, SequenceColumnCreate}
 import io.scalaland.chimney.Transformer
+import io.scalaland.chimney.dsl.TransformerConfiguration
 
 // scalastyle:off
 package object v1 {
+  implicit val chimneyConfiguration =
+    TransformerConfiguration.default.enableMethodAccessors.enableDefaultValues
```

Comment:
Both `MethodAccessors` and `DefaultValues` were made opt-in as users considered them to be unsafe defaults.

Reply:
@MateuszKubuszok what was the need for `MethodAccessors`, was the default for it changed in chimney moving to v1?
Going to 0.7.5 required only `DefaultValues` to compile https://github.com/cognitedata/cdp-spark-datasource/pull/794/files#diff-0b0d81435732de336a95503c47d48d0fffe6a09474a06d0475d0626a7ce3d14bR11
Going to v1 also doesn't seem to need it https://github.com/cognitedata/cdp-spark-datasource/pull/941/files (copy of this PR updated for latest master branch)

Reply:
I added it when I saw the error that there is no accessor with a particular name - however right after that I noticed that virtually all derivations in your codebase rely on default values rather than method accessors. It can be skipped.